src/prefect/tasks/gsheets/gsheets.py

```diff
@@ -0,0 +1,152 @@
+from prefect.utilities.tasks import defaults_from_attrs
+import gspread
+from typing import Dict, List, Tuple, Union, Any, TypedDict, Callable
+import prefect
+from prefect import task, Flow, Parameter, Task
+from prefect.tasks.secrets.base import SecretBase
+from prefect.tasks.secrets import PrefectSecret
+import pathlib
+from pathlib import Path

```

Comment:
Looks like there are some unused imports in here that our static analysis job is failing on; it's suggesting:
```
-from typing import Dict, List, Tuple, Union, Any, TypedDict, Callable

-import prefect

-from prefect import task, Flow, Parameter, Task

-from prefect.tasks.secrets.base import SecretBase

-from prefect.tasks.secrets import PrefectSecret

+from typing import Any, Callable, List, TypedDict, Union

+from prefect import Task, task
```

Reply:
Removed!